python/ray/data/tests/test_stats.py

```diff
@@ -1062,9 +1099,10 @@ def test_summarize_blocks(ray_start_regular_shared, stage_two_block):
     calculated_stats = stats.to_summary()
     summarized_lines = calculated_stats.to_string().split("\n")
 
+    block_max_time = max(m.exec_stats.wall_time_s for m in block_meta_list)
```

Comment:
It would be good to test against the actual output that we expect instead of internal block values (which may not be right either).

Reply:
Since the test uses `BlockMetadata` created with pre-determined parameters, and we are not reading/manipulating Datasets here, what would be the appropriate "actual output" to compare to in this case? Do you mean I should hardcode the expected value here?

Reply:
Yeah, I was thinking something like checking it against how much wall time has passed (so it is more about matching user expectations). For example, we could do something like this:
- Record start time
- Run a Dataset with N tasks that sleep(T), using N / 2 CPUs
- Check reported stage time is >=2*T and <= actual time passed since start